docs/core/extensions/best-practices-for-developing-world-ready-apps.md

```diff
@@ -42,6 +42,11 @@ This section describes the best practices to follow when developing world-ready
 
 1. If a security decision is based on the result of a string comparison or case change operation, use a culture-insensitive string operation. This practice ensures that the result is not affected by the value of `CultureInfo.CurrentCulture`. See the ["String Comparisons that Use the Current Culture"](../../standard/base-types/best-practices-strings.md#string-comparisons-that-use-the-current-culture) section of [Best Practices for Using Strings](../../standard/base-types/best-practices-strings.md) for an example that demonstrates how culture-sensitive string comparisons can produce inconsistent results.
 
+1. For any element being used for interchange (eg, a field in a JSON document in an API call) or storage, use the <xref:System.Globalization.CultureInfo?displayProperty=InvariantCulture>; additionally, you should explicitly specify a roundtrip format (such as the [`"O"`, `"o"` date-time format specifier](../../standard/base-types/standard-date-and-time-format-strings.md#the-round-trip-o-o-format-specifier)).  Although the format strings for the invariant culture are stable and unlikely to change, specifying an explicit format string helps to clarify the intent of your code.
+    - For date/time elements, consider the advice and observations in Jon Skeet's post [_Storing UTC is not a silver bullet_](https://codeblog.jonskeet.uk/2019/03/27/storing-utc-is-not-a-silver-bullet/).
+
+1. Globalization data [is not stable](https://learn.microsoft.com/en-us/archive/blogs/shawnste/culture-data-shouldnt-be-considered-stable-except-for-invariant), and you should write your application and its tests with this in mind.  It is updated several times a year through host OS channels on all supported platforms.  This data is not maintained by or distributed with the runtime, nor (outside of legacy Windows applications) directly by the host OS.
```

Comment:
```suggestion
1. Globalization data [is not stable](https://learn.microsoft.com/en-us/archive/blogs/shawnste/culture-data-shouldnt-be-considered-stable-except-for-invariant), and you should write your application and its tests with this in mind.  It is updated several times a year through host OS channels on all supported platforms.  This data is typically not distributed with the runtime.
```
For example, iOS and wasm shipped the globalization data too, it has changed recently with hybrid globalization. I would not into enumerating all past and present cases where the globalization data ships with the runtime or the application.